### PR TITLE
FACINGS - NFT Collection Manager - Milestone 2 delivery

### PR DESCRIPTION
**Pull request by robrigo**
# Milestone Delivery Checklist

- [x] The [milestone-delivery-template.md](https://github.com/eosnetworkfoundation/grant-milestones/blob/master/deliveries/milestone-delivery-template.md) has been copied and updated.
- [x] The [invoice form](https://forms.gle/wLuAzXKa9qYrZQob9) has been filled out for this milestone.
- [x] This pull request is being made by the same account as the accepted application.
- [x] In case of acceptance, the payment will be transferred to the EOS account name provided in the application.
- [x] The delivery is according to the [Guidelines for Milestone Deliverables](https://github.com/eosnetworkfoundation/grant-framework/blob/master/docs/milestone-deliverables-guidelines.md).

Link to the application pull request: https://github.com/eosnetworkfoundation/grant-framework/pull/36 < please fill this in with the PR number of your application.

### CODE REVIEW

**Review by DenisCarriere — APPROVED** [review]
Tested latest release
- Install
- Build
- Deploy dev
- Tested creating Collection

**Review by nsjames — APPROVED** [review]

File: deliveries/facings-nft-collection-manager-2.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+| -----: | ------------------- | ------------------------------------------------------------------ | ----------------------------------------------------------------------- |
+| 0a.    | License             | https://github.com/FACINGS/collection-manager/blob/main/LICENSE    | GPLv3                                                                   |
+| 0b.    | Documentation       | https://github.com/FACINGS/collection-manager/blob/main/README.md  | Project overview and "Getting Started" guide, deployment instructions, inline documentation. |
+| 0c.    | Testing Guide       | https://github.com/FACINGS/collection-manager/blob/main/testing-guide.md | Documentation for a walkthrough of the UI as described.           |
+| 0d.    | Docker              | https://github.com/FACINGS/collection-manager/blob/main/Dockerfile | Instructions are included in aforementioned "Getting started" guide.    |
+| 0e.    | Article             | https://docs.google.com/document/d/1hJuVdnpm0AIhnZWykrk2yEaWcA1-6Y3bDortwQ48rI4/edit | (Soon to be) published article describing the work we performed as part of the grant, introduce our app, and highlight the features which make it valuable. |
```
Comment (larryk85) [inline deliveries/facings-nft-collection-manager-2.md:20]: I don't have access to this document.

**Review by wanderingbort — APPROVED** [review]

File: deliveries/facings-nft-collection-manager-2.md (hunk 1 above)
Comment (robrigo) [reply deliveries/facings-nft-collection-manager-2.md:20]: We have a review draft of the article [here](https://docs.google.com/document/d/1SFmLEN4stBhDtoKXYX2nS8A0o0eHz1z99QUELm3F5D0/edit#). This link is referenced from the Article link in the above table. Thanks for the thorough review. )

Comment (Keirmon) [reply deliveries/facings-nft-collection-manager-2.md:20]: > I don't have access to this document.

Let us know if you are still having any difficulty, Everything should be up and all links and posts will go live on approval. Thank you.

**Review by larryk85 — APPROVED** [review]
LGTM

**Comment by stephenpdeos** [comment]
All reviewers approved cc @tedcahalleos

**Pull request merged**